lib/profileAgents.js

```diff
@@ -96,7 +99,7 @@ exports.create = async ({accountId, profileId, token} = {}) => {
   const key = await keystoreAgent.generateKey(
     {type: 'Ed25519VerificationKey2018', kmsModule: KMS_MODULE});
   // 5. Use the zCap key to create the DID for the profile agent
-  await utils.setKeyId({key});
+  key.id = await utils.computeKeyId({key, didMethod, didOptions});
```

Comment:
Shouldn't this always be using `did:key`? What's the use case for having the Profile Agent use `did:v1`. If we do want to support configurable DID Methods for Profile Agents, then we should ensure that dependent libraries are explicitly choosing the methods during creation. We must not start the pattern of selecting a `did:v1` based Profile, then that automatically creates a `did:v1` Profile Agent.

Reply:
+1, profile agents only need did:key for now.

Reply:
@gannan08 @dlongley I think the best thing to do in this case is - let's leave the options available in this lib, and then require `did:key` in an upstream library, here: https://github.com/digitalbazaar/bedrock-profile-http/pull/7/files#diff-1c0f1c434b17b7f8795d44a51a14320aR75

(That way, we can change that in `bedrock-profile-http` without needing to re-release this lib.)

Reply:
I think it's more likely to get misused than used properly at this point. We're not sure if we ever want to support it -- and it's not hard to add it back if we need it, so I think we should remove it.

Reply:
Hahahaha, you got it, removing.